docs/security/customize-user-lockout.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Customize max login attempts and duration of user lockout
+description: Use Clerk to limit the number of times a user can attempt to log in before they are locked out of their account and how long such lockouts last.
+---
+
+# Customize the max attempts before getting locked out and duration of lock out
+
+<Callout type="info">
+This feature is applicable to user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) or [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
+</Callout>
+
+1. In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **User & Authentication > [Attack Protection](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection)**.
+2. To change the number of failed attempts before a user is locked out, under **Maximum attempts,** select "Custom attempts" and enter a new number of failed attempts allowed. (The default is 100 attempts.)
+3. To change the duration, under **Duration,** select "Custom duration" and select the unit of time (minutes/hours/days/years) and enter the number of units you want lockouts to last.
+4. Select "Apply changes" to apply your settings.
+
+<Images
+width={1512}
+height={914}
+src="/docs/images/security/userlock_custom-attempts-and-duration.png"
+alt=" "
+/>
+
+## Lock a user account forever until an admin unlocks the account
+
+1. In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **User & Authentication > [Attack Protection](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection)**.
+2. Under **Duration,** select "Indefinitely."
+3. Select "Apply changes" to apply your settings.
+
+<Images
+width={1512}
+height={914}
+src="/docs/images/security/userlock_indefinite.png"
+alt=" "
+/>
```

Comment:
Same feedback from the comment before about the text matching the screenshot - let's make sure the copy reflects the exact text that is used in the Dashboard. So if the Dashboard says "Lockout duration", we should say "Lockout duration" instead of "Duration", to help accurately and clearly navigate users.